src/app.jsx

```diff
@@ -45,6 +47,10 @@ class App extends React.Component {
       this.setState({countdown: this.state.sessionLength});
     }
 
+    if (!this.state.focusTimestamp) {
```

Comment:
at start of function, reset the focusTimestamp to empty array, reset the refocusCount to 0